projects/ion/src/lib/core/types/triple-toggle.ts

```diff
@@ -0,0 +1,21 @@
+import { EventEmitter } from '@angular/core';
+import { SafeAny } from '../../utils/safe-any';
+import { Size } from './button';
+import { IconType } from './icon';
+
+export interface TripleToggleSetting {
+  value: SafeAny;
+  label?: string;
+  tooltip?: string;
+  icon?: IconType;
+  rightSideIcon?: boolean;
+  selected?: boolean;
+}
+
+export interface TripleToggleProps {
+  value?: SafeAny;
+  disabled?: boolean;
+  size?: Size;
+  configuration?: TripleToggleSetting[];
```

Comment:
`@Input() toggleOptions: [toggleOption, toggleOption, toggleOption];`
If typed like this, it would solve the issue with the number of options informed?